src/window_selectable.cpp

```diff
@@ -64,11 +64,11 @@ int Window_Selectable::GetPageItemMax() {
 
 Rect Window_Selectable::GetItemRect(int index) {
 	Rect rect = Rect();
-	rect.width = (contents->GetWidth()) / column_max - 4;
+	rect.width = (contents->GetWidth()) / column_max - 8;
 	rect.height = 12;
 	rect.x = index % column_max * rect.width;
 	if (rect.x > 0){
-		rect.x += 8;
+		rect.x += 16;
```

Comment:
Pardon me for reviewing this after lack of sleep. Took another look and this actually appears wrong.

If you take a screenshot of the item window and zoom in, you'll the item text in EasyRPG is to the right by 1 whole pixel.

Also this doesn't actually work well if the `column_max > 2`

Reply:
Fixed.